packages/json-schemas/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "1.0.1-rc.5",
+        "changes": [
+            {
+                "note": "Update incorrect relayer api fee recipients response schema",
```

Comment:
Can you also create a 2.0.0 version of this file: https://github.com/0xProject/0x-monorepo/blob/development/packages/website/md/docs/json_schemas/1.0.0/schemas.md

And add an entry to `sectionNameToMarkdownByVersion` in: https://github.com/0xProject/0x-monorepo/blob/development/packages/website/ts/containers/json_schemas_documentation.ts So it renders when someone is looking at a >1.0.0 version?

Reply:
Is this blocking the merge? I have another ticket related to SRA documentation and I would prefer to do it there.

Reply:
Yes it's blocking.